pyproject.toml

```diff
@@ -0,0 +1,84 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) Contributors to the OpenEXR Project.
+
+[build-system]
+requires = ["scikit-build-core==0.8.1", "pybind11"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "Imath"
+dynamic = ["version"]
+
+description="Python bindings for the Imath  a basic, light-weight, and efficient C++ representation of 2D and 3D vectors and matrices and other simple but useful mathematical objects, functions, and data types common in computer graphics applications, including the “half” 16-bit floating-point type."
+readme = "README.md"
+authors = [
+  { name="Contributors to the OpenEXR project", email="info@openexr.com" },
+]
+requires-python = ">=3.7"
+
+[project.urls]
+"Homepage" = "https://imath.readthedocs.io/"
+"Source" = "https://github.com/AcademySoftwareFoundation/Imath"
+"Bug Tracker" = "https://github.com/AcademySoftwareFoundation/Imath/issues"
+
+[project.optional-dependencies]
+test = ["pytest"]
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+addopts = ["-ra", "--showlocals", "--strict-markers", "--strict-config"]
+xfail_strict = true
+log_cli_level = "INFO"
+filterwarnings = [
+  "error",
+]
+testpaths = ["tests"]
+
+[tool.scikit-build]
+wheel.expand-macos-universal-tags = true
+sdist.exclude = [".github", "src/**Test", "website", "share"]
+
+# Only build the PyOpenEXR (cmake --build --target PyOpenEXR).
+cmake.targets = ["Imath"]
+# Only install the "python" component (cmake --install --component python).
+# This makes sure that only files marked as "python" component are installed.
+install.components = ["python"]
+# strip debug symbols
+install.strip = true
+
+# Enable experimental features if any are available
+# In this case we need custom local plugin to get
+# the project version from cmake.
+experimental = true
+metadata.version.provider = "PyImath"
+metadata.version.provider-path = "./src/python"
+
+
+[tool.scikit-build.cmake.define]
+IMATH_INSTALL = 'OFF'
+IMATH_BUILD_PYTHON = 'ON'
+IMATH_BUILD_EXAMPLES = 'OFF'
```

Comment:
Double check which of these variables actually exist.  I think
```
IMATH_BUILD_EXAMPLES
IMATH_BUILD_TOOLS
IMATH_INSTALL_TOOLS
IMATH_FORCE_INTERNAL_DEFLATE
IMATH_FORCE_INTERNAL_IMATH
IMATH_TEST_LIBRARIES
```

don't exist, that was copy/paste from OpenEXR.

Reply:
thanks was trying to figure out, thanks